openapi/spec/resources/sessions/record_del.yaml

```diff
@@ -15,12 +15,24 @@ delete:
     '200':
       $ref: ../../shared/responses/OkResponse.yaml
     '400':
-      $ref: ../../shared/responses/BadRequestResponse.yaml
+      description: Bad Request
+      content:
+        application/json:
+          schema:
+            type: string
```

Comment:
Couldn't this to be added to the shared response file?

Reply:
I guess no, because if I try to make something like snippet below, the code generate is invalid.

```openapi
    '403':
      description: Forbidden
      $ref: ../../shared/responses/JSONStringResponse.yaml
```
![Screenshot 2021-09-27 at 16-51-34 Screenshot](https://user-images.githubusercontent.com/23109089/134975778-6d518b12-89f9-4144-87cf-0baed4deb2e6.png)

```openapi
# JSONStringResponse.yaml
content:
  application/json:
    schema:
      type: string
```

Reply:
It should use  a shared definition. @gustavosbarreto what do you think?